### PR TITLE
EZP-31792: Redesign UDW

### PR DESCRIPTION
**Pull request by lucasOsti**
Jira ref: https://jira.ez.no/browse/EZP-31792

### CODE REVIEW

File: src/bundle/Resources/public/scss/_buttons.scss [hunk 1]
```diff
@@ -247,3 +247,7 @@
         fill: $ibexa-color-primary;
     }
 }
+
+button:focus {
+    outline: 0;
+  }
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_buttons.scss:253]: indent

File: src/bundle/Resources/public/scss/ui/modules/universal-discovery/_content.create.button.scss [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
         padding: calculateRem(4px);
-        width: calculateRem(34px);
-        height: calculateRem(34px);
-        display: flex;
+        color: $ibexa-color-primary;
+        text-decoration: none;
+    }
+
+    .ez-icon {
+        fill: $ibexa-color-primary;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/ui/modules/universal-discovery/_content.create.button.scss:12]: I think the primary color deserves to have an icon modifier.

**Comment by GrabowskiM** [comment]
Fix PR name

File: src/bundle/Resources/views/themes/admin/content/edit/edit.html.twig [hunk 3]
```diff
@@ -6,79 +6,28 @@
...
+{% block breadcrumbs %}
+    {% include '@ezdesign/ui/breadcrumbs.html.twig' with { items: [
+        { value: content_type.name }
+    ]} %}
+{% endblock %}
+
+{% block page_title %}
+    {% set content_name = content ? content.name : 'new_content_item'|trans({'%contentType%': content_type.name})|desc('New %contentType%') %}
+    {% include '@ezdesign/content/page_title_edit.html.twig' with { 
+        action_name: 'Editing'|trans|desc('Editing'),
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/content/edit/edit.html.twig:18]: translation id shouldn't be capitalized, I think

File: src/bundle/Resources/public/scss/ui/modules/universal-discovery/_content.create.button.scss (hunk 2 above)
Comment (lucasOsti) [reply src/bundle/Resources/public/scss/ui/modules/universal-discovery/_content.create.button.scss:12]: We have modifier `ez-icon--primary`

**Pull request merged**